script/configs/allowed_pinned_deps.yaml

```diff
@@ -0,0 +1,24 @@
+# The list of external dependencies that are allowed as unpinned dependencies.
+# See https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#Dependencies
+#
+# All entries here should have an explanation for why they are here.
+
+# TODO(stuartmorgan): Eliminate this in favor of standardizing on
+# mockito.
+- mocktail
+
+# Legacy allowances, for dependencies that predate the tooling.
+# TODO(stuartmorgan): Audit these. See
+# https://github.com/flutter/flutter/issues/122713
+- jwt_decoder
```

Comment:
`jwt_decoder` should be easy to remove. I checked because I remember removing it a while back... but looks like I forgot to clean-up the package **example** 🤦

Reply:
(I'm currently working on this package, I'll sneak a quick fix for this after you land the tooling change)

Reply:
Opportunistically removed this dep in this PR, resolving: https://github.com/flutter/packages/pull/3432